content/docs/hello-world.md

```diff
@@ -17,7 +17,7 @@ ReactDOM.render(
 
 It displays a heading saying "Hello, world!" on the page.
 
-[](codepen://hello-world)
+**[Try it on CodePen](https://codepen.io/gaearon/pen/rrpgNB?editors=0011)**
```

Comment:
`?editor=0011`
This query is added in all the links because I had seen other CodePen links in React Documentation there too this type of query exist, it just minimizes the HTML and CSS tab and opens a console when the CodePen is opened

Reply:
Since these are basic examples we might not need console. Also we can go with `?editors=1010` to open `HTML`, `JS` and show output alone? Your thoughts ?

Reply:
Alright, I agreed, I will just keep HTML, JS, and output and won't open the console tab 👍🏻